orttraining/orttraining/python/ort_trainer.py

```diff
@@ -8,6 +9,7 @@
 import torch.onnx
 import onnxruntime as ort
 from distutils.version import LooseVersion
+from .checkpointing_utils import list_checkpoint_files, Combine_Zero_Checkpoint
 
 DEFAULT_OPSET_VERSION = 10
 
```

Comment:
filename is being computed in several places, which can be tricky when changes are needed. One alternative is to create file masks in one place and use it everywhere

```python3
def get_checkpoint_name(partition_optimizer, prefix,world_rank=None, world_size=None):
    SINGLE_CHECKPOINT_FILENAME='{prefix}.ort.pt'
    MULTIPLE_CHECKPOINT_FILENAME='{prefix}.ZeRO.{world_rank}.{world_size}.ort.pt'
    if partition_optimizer:
        filename=MULTIPLE_CHECKPOINT_FILENAME.format(prefix=prefix, world_rank=world_rank,     world_size=world_size)
    else:
        filename=SINGLE_CHECKPOINT_FILENAME.format(prefix=prefix)
````